docs/tutorial/connecting.rst

```diff
@@ -11,10 +13,21 @@ First import the instrument of interest. ::
 
     from pymeasure.instruments.keithley import Keithley2400
 
-Then construct an object by passing the GPIB address. For this example we connect to the instrument over GPIB (using VISA) with an address of 4. See the :ref:`adapters <adapters>` section below for more details. ::
+Then construct an object by passing the GPIB address. For this example we connect to the instrument over GPIB (using VISA) with an address of 4::
```

Comment:
The first sentence is valid for all VISA addresses, afterwards adapted to the example.
```suggestion
Then construct an object by passing the VISA address. For this example we connect to the instrument over GPIB (using VISA) with an address of 4::
```